Assets/MXR.SDK/Plugins/Android/AdminAppMessengerManager.java

```diff
@@ -212,7 +212,7 @@ public boolean sendMessage(int what, String jsonString) {
         return true;
     }
 
-    private ComponentName getInstalledAdminAdminServiceComponent() {
+    public ComponentName getInstalledAdminAdminServiceComponent() {
```

Comment:
Making this public lets users depend on the package name info explicitly, which is something we don't want. https://blog.axway.com/learning-center/apis/api-design/hyrums-law-and-api-design-and-management#:~:text=Here%20is%20what%20Hyrum's%20law,be%20depended%20on%20by%20somebody. 

Can we instead make a public method that calls this function and checks for null?